lib/log_vty.c

```diff
@@ -169,6 +169,38 @@ void log_show_syslog(struct vty *vty)
 			zlog_progname);
 }
 
+DEFPY(send_log,
+      send_log_cmd,
+      "send log [severity <emergency|alert|critical|error|warning|notice|info|debug>$levelarg] LINE...",
```

Comment:
In all commands and [documentation](https://docs.frrouting.org/projects/dev-guide/en/latest/logging.html#log-levels) we refer to these as "log levels" while "severity" don't show anywhere.

I suggest making the command 
```
send log [level LEVEL]...
```
```

Reply:
I was specifically not using level here. We have mixed "priority" `log record-priority` and level `log stdout LEVEL` both of which use home-grown non-std keywords `emergencies, alerts, critical, errors, warnings, notifications, informational, or debugging`.

I'm trying to move us to the industry standard syslog severity notation. I have another PR which adds `log record-severity` so that the output to the log is using standard syslog severity tags, which auto-stylizing/coloring pagers as well as most users scripts understand.

Currently we output lower-case homegrown tags like `errors: foo` and `debugging: foo`. `log record-severity` is going to output industry standard tags `ERROR: `, `DEBUG: ` ...

This change is going along with that.

Reply:
I suppose we could support both.

Reply:
The commercial router variant of this takes only an integer value from 0-7

Reply:
Actually nm we can make this change later if we want to. I'll just use level and the current home-grown keyword values